anta/cli/get/utils.py

```diff
@@ -214,6 +215,56 @@ def create_inventory_from_ansible(inventory: Path, output: Path, ansible_group:
     write_inventory_to_file(ansible_hosts, output)
 
 
+def create_inventory_from_netbox(nb_instance: str, output: Path, token: str, platform: str = "Arista EOS", site: str | None = None, verify: bool = False) -> None:
+    """Fetch devices from NetBox filtered by a specific platform.
+
+    Parameters
+    ----------
+    nb_instance
+        The NetBox API instance.
+    output
+        ANTA inventory file to generate.
+    token
+        The token used to authenticate to the NetBox instance.
+    platform
+        The platform to filter devices by.
+    site
+        The site to filter devices by.
+    verify
+        Verify the SSL certification of the NetBox instance.
+
+    """
+    session = requests.session()
+    session.verify = verify
+    try:
+        import pynetbox
+    except ImportError as e:
+        logging.error(e)
```

Comment:
Lets make a better error message telling what to instal `pip install anta[netbox]`?